frontend/components/joinUsSection.vue

```diff
@@ -0,0 +1,92 @@
+<template>
```

Comment:
Component name should be always pascal-cased or kebab-cased. You can base on official [styleguide](https://vuejs.org/v2/style-guide/).

Reply:
Already done, thanks.

Reply:
Please resolve conversation if everything is ok :)